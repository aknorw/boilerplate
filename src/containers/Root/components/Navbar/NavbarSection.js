import styled from 'styled-components'

export default styled.div`
  display: inline-flex;
  align-items: baseline;
  font-size: 1em;
  > *:not(:last-child) {
    margin-right: .5rem;
  }
`
