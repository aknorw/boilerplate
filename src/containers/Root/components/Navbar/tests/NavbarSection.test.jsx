import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import NavbarSection from '../NavbarSection'

describe('<NavbarSection />', () => {
  it('should be defined', () => {
    expect(NavbarSection).toBeDefined()
  })
  it('should render correctly', () => {
    const tree = shallow(
      <NavbarSection />,
    )
    expect(tree).toMatchSnapshot()
  })
  it('should have style rules', () => {
    const tree = shallow(
      <NavbarSection />,
    )
    expect(tree).toHaveStyleRule('display', 'inline-flex')
    expect(tree).toHaveStyleRule('align-items', 'baseline')
    expect(tree).toHaveStyleRule('font-size', '1em')
  })
})
