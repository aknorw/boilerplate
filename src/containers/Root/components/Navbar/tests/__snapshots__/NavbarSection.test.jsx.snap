// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarSection /> should render correctly 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  font-size: 1em;
}

.c0 > *:not(:last-child) {
  margin-right: .5rem;
}

<div
  className="c0"
/>
`;
